### PR TITLE
Version: 1.0.0

### PR DESCRIPTION
**Pull request by kmuehlbauer**

### CODE REVIEW

**Comment by kmuehlbauer** [comment]
@conda-forge-admin, please rerender.

**Comment by kmuehlbauer** [comment]
@conda-forge-admin, please rerender.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,13 +16,15 @@ build:
 requirements:
     build:
         - python
+        - setuptools
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Try `pip` here and update the script line to `script: python -m pip install --no-deps --ignore-installed .`

See https://conda-forge.org/docs/meta.html#use-pip

Comment (kmuehlbauer) [reply recipe/meta.yaml]: @ocefpaf To fast for me today. I have to add `pip`. And then where should I put these line?

Comment (kmuehlbauer) [reply recipe/meta.yaml]: @ocefpaf Got it, finally :+1:

**Pull request merged**